src/main/java/gtPlusPlus/core/recipe/RecipesGregTech.java

```diff
@@ -721,6 +721,49 @@ private static void extruderRecipes() {
 
     private static void blastSmelterRecipes() {
 
+        // Enderium
+        GTValues.RA.stdBuilder()
+            .itemInputs(
+                GTUtility.getIntegratedCircuit(2),
+                Materials.Tin.getDust(2),
+                Materials.Platinum.getDust(1),
+                Materials.Silver.getDust(1),
+                Materials.Thaumium.getDust(2),
+                Materials.EnderPearl.getDust(2))
+            .fluidOutputs(Materials.Enderium.getMolten(8 * 144))
+            .eut(TierEU.RECIPE_EV)
+            .duration(20 * SECONDS + 1 * MINUTES)
+            .addTo(alloyBlastSmelterRecipes);
+
+        // Signalium
+        GTValues.RA.stdBuilder()
+            .itemInputs(
+                GTUtility.getIntegratedCircuit(2),
+                Materials.AnnealedCopper.getDust(30),
+                Materials.Ardite.getDust(10),
+                Materials.Redstone.getDust(50))
+            .fluidOutputs(GGMaterial.signalium.getMolten(5 * 144))
+            .eut(TierEU.RECIPE_LuV)
+            .noOptimize()
+            .duration(5 * MINUTES)
+            .addTo(alloyBlastSmelterRecipes);
+
+        // Lumiium
+        GTValues.RA.stdBuilder()
+            .itemInputs(
+                GTUtility.getIntegratedCircuit(2),
+                Materials.Tin.getDust(10),
+                Materials.Iron.getDust(10),
+                Materials.Copper.getDust(2),
+                Materials.Silver.getDust(8),
+                GGMaterial.lumiinessence.get(OrePrefixes.dust, 10),
+                Materials.Glowstone.getDust(10))
+            .fluidOutputs(GGMaterial.lumiium.getMolten(5 * 144))
+            .eut(TierEU.RECIPE_LuV)
+            .noOptimize()
+            .duration(4 * MINUTES)
```

Comment:
```suggestion
            .duration(5 * MINUTES)
```
Lumiium should have the same length as signalium, since both have the same duration in ebf